### PR TITLE
Add sendmail SMTP configuration option

### PR DESCRIPTION
**Pull request by bbodenmiller**

### CODE REVIEW

**Comment by cwarnermm** [comment]
@coltoneshaw - Are you open to reviewing this documentation PR for technical accuracy?

**Review by coltoneshaw — CHANGES_REQUESTED** [review]
I think we can most likely move this up to be a suggestion under the above bullet, like below. Since that above bullet technically includes it. However, technically there are no issues with using localhost and port 25.

```
If you don't have an SMTP service, here are simple instructions:
- Amazon Simple Email Service (SES):
- Local ``sendmail`` can be used by using **Server Name** ``127.0.0.1``
```

**Comment by cwarnermm** [comment]
Thanks, @coltoneshaw. Great feedback! I agree with incorporating this new content into the bullet point that reads "If you don't have an SMTP service, here are simple instructions..."

@bbodenmiller - Are you open to updating your PR with this feedback? If you'd prefer, I'm can help complete this update.

**Comment by bbodenmiller** [comment]
Might get done faster if you make the tweak.

**Review by coltoneshaw — APPROVED** [review]

**Comment by coltoneshaw** [comment]
@cwarnermm / @bbodenmiller - I adjusted the requested structure. Let me know if it flows okay.

File: source/install/smtp-email-setup.rst [hunk 1]
```diff
@@ -13,40 +13,41 @@ To enable email, configure an SMTP email service as follows:
...
+                Framework <https://en.wikipedia.org/wiki/Sender_Policy_Framework>`__
+                (SPF) and/or `Domain Keys Identified
+                Mail <https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail>`__
+                (DKIM) for your email domain.
+
+            4. Choose a sender address like ``mattermost@example.com`` and
+                click ``Send a Test Email`` to verify setup is working
+                correctly.
+	    
+        - By local ``sendmail`` can be used by using **Server Name** ``127.0.0.1`` with **Port** ``25``
```
Comment (bbodenmiller) [inline source/install/smtp-email-setup.rst:42]: @coltoneshaw `by` seems a bit weird here.

Comment (cwarnermm) [reply source/install/smtp-email-setup.rst:42]: Agreed. Looks like "by" isn't needed.

**Review by cwarnermm — APPROVED** [review]
This is a great documentation update - thank you so much!

**Pull request merged**